### PR TITLE
Rebase govuk-frontend-3 onto master

### PR DESCRIPTION
**Pull request by domoscargin**
Submitted a PR to enable review.

Idea is to reset govuk-frontend-3 to this branch.

### CODE REVIEW

File: package/package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "digitalmarketplace-govuk-frontend",
   "description": "Digital Marketplace GOV.UK Frontend contains the code you need to start building a user interface for Digital Marketplace.",
-  "version": "2.0.0",
+  "version": "2.0.0-govuk-frontend-3-r2",
```
Comment (lfdebrux) [inline package/package.json:4]:
```suggestion
  "version": "2.0.0-govuk-frontend-3-r3",
```

Comment (lfdebrux) [reply package/package.json:4]: This should probably be a major version bump still

**Review by lfdebrux — APPROVED** [review]
This looks good to me

**Comment by domoscargin** [comment]
Closing this PR and resetting govuk-frontend-3 branch.

**Pull request closed without merging**